.github/ISSUE_TEMPLATE.md

```diff
@@ -1,33 +1,32 @@
 <!--
-Before filing an issue, please check our Frequently Asked Questions:
+Before filing an issue, please verify the issue still occurs with the latest version of Jib.
```

Comment:
Should we also leave the FAQ thing in there?

Reply:
I didn't analyze the incoming issues, but it seems most that are FAQ'd are relating to registry errors.  Which is covered by the next part.